### PR TITLE
Create Global Params For Fee Estimator Congestion and Priority Params

### PR DESCRIPTION
**Pull request by tholonious**
Create Global Params For Fee Estimator Congestion and Priority Params

Implement encode and decode

Rename

### CODE REVIEW

**Comment by tholonious** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1258?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1258** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1258?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1256** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1256?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1252** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1252?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1255](https://github.com/deso-protocol/core/pull/1255) <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1255?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#1253** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1253?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tholonious and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/block_view.go [hunk 1]
```diff
@@ -3491,6 +3491,74 @@ func (bav *UtxoView) _connectUpdateGlobalParams(
...
+					"_connectUpdateGlobalParams: MempoolPastBlocksPriorityPercentileBasisPoints must be <= %d",
+					MaxBasisPoints,
+				)
+			}
+			if bytesRead <= 0 {
+				return 0, 0, nil, fmt.Errorf(
+					"_connectUpdateGlobalParams: unable to decode MempoolPastBlocksPriorityPercentileBasisPoints as uint64",
+				)
+			}
+			newGlobalParamsEntry.MempoolPastBlocksPriorityPercentileBasisPoints = val
```
Comment (lazynina) [inline lib/block_view.go:3560]: I probably would have named this `PastBlocksPriorityPercentileBasisPoints` instead of `MempoolPastBlocksPriorityPercentileBasisPoints` or would have used a prefix like `FeeEstimator` for these so we would have `FeeEstimatorPastBlocksPriorityPercentileBasisPoints`. That feels a bit wordy tho. I just think using `Mempool` as a prefix is a bit confusing, but these were the names DH was using so if that's what he wants, it's fine by me.

**Review by lazynina — APPROVED** [review]

**Comment by tholonious** [comment]
### Merge activity

* **Apr 22, 10:39 PM EDT**: @tholonious started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/1258).
